Vagrantfile

```diff
@@ -126,6 +126,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     # Start an OpenShift cluster
     # Currently this only works with the (default) VirtualBox provider.
 
+    # Tag configuration as stale when provisioning a dev cluster to
+    # ensure that nodes can wait for fresh configuration to be generated.
+    if ARGV[0] =~ /^up|provision$/i and not ARGV.include?("--no-provision")
+      system('test -d ./openshift.local.config && touch ./openshift.local.config/.stale')
```

Comment:
Then I feel like I'm missing something. The scenario is a local vagrant deployment which is mounting a config dir which already contains a config, which you are then marking stale so nodes will wait, but I don't see anything that will remove the stale marker. If you start a local vagrant cluster and mount in a config dir that already contains config data, the master will overwrite bits of it on start, but the stale marker will be left, and the nodes will hang forever, right?